packages/dds/cell/src/test/cell.spec.ts

```diff
@@ -189,10 +192,12 @@ describe("Cell", () => {
 				);
 				await cell2.load(services);
 
+				key = cell2.getAttribution();
+
 				assert.equal(
-					cell2.getAttribution(),
-					undefined,
-					"the second cell should not have valid attribution",
+					key?.type,
```

Comment:
local keys should never be serialized--summarization process should never occur with local ops (unless detached, but that's why there's a different key for detached attribution). I'd actually prefer to test that attempting to summarize a local key throws an assert so we can be sure we don't accidentally write it into document data. Then this test can be changed to not bother loading a second client.